locales/en/index.yml

```diff
@@ -4054,6 +4054,8 @@ transaction:
     totalFee: Il totale comprende
     totalFeePsp: "di commissione, applicata da {{pspName}}."
     totalFeeNoPsp: "di commissione, applicata dal gestore della transazione (PSP)."
+    bannerImported:
```

Comment:
This part must be in English

Reply:
The English copy should be provided by either @FrancescaMariaTIberti or @thisisjp. Since there are other strings that also need to be translated into English in addition to what's been added in this PR, what do you think about opening a separate PR for the English copy once @FrancescaMariaTIberti or @thisisjp provide all the translations? Unfortunately, we can't do a direct, word-for-word translation, so we'll need their review to ensure the text fits both in terms of meaning and length.

Reply:
Ok cool! We will solve the translation issue in other PR